storage/database/src/main/java/de/danoeh/antennapod/storage/database/ReleaseScheduleGuesser.java

```diff
@@ -62,9 +66,9 @@ private static void addTime(GregorianCalendar date, long time) {
         date.setTime(new Date(date.getTime().getTime() + time));
     }
 
-    private static void addUntil(GregorianCalendar date, List<Integer> days) {
+    private static void addUntil(GregorianCalendar date, long amount, List<Integer> days) {
```

Comment:
Maybe `addTimeUntilOnAllowedDay` and calling the parameter `allowedDays`? That would be a bit easier to understand for future readers (I know that you did not invent the current name)